README.rst

```diff
@@ -3,8 +3,16 @@ Chortle
 IoT based chore tracking dashboard  
 We designed this to leverage AWS IoT buttons and display a chart of household chores.  
 
-When you press a button it will mark a chore (for example clean the cat box) as done  
+Chores are managed using one of three strategies:
```

Comment:
This needs a little bit of revision. A few notes:

- Line breaks are not showing up properly. Make sure a line ends in two spaces to get the doc to display a line break. README format is RST.

- Chore types are incorrect. We currently use "toggle" (for what you are calling modal), and "modal" (for what you are calling "phased").

- Maybe we should have different verbiage than "open"? In the doc comments I used "green" and "red", dunno if that's better. Open could be fine.

- We should add an example Toggle chore (emptying the dishwasher? cleaning the kitchen?), and perhaps document that toggle chores can optionally have a timeout between the button press and the chore going red.

- Periodic chores currently don't work in the way described. A periodic chore will not go red until the chore has not been done within the needed period (when the timer runs out completely). It's just that the timer itself does not display, until some constant time before it's about to run out (currently 3 hours). Displaying the timer at a time determined as a percentage of the period is a future enhancement.

- Modal chores don't quite work this way. If you have a dependency relationship between chores A and B, it's not that closing A opens B, it's actually that opening B closes A. It's a "pull" rather than a "push", which is intentional because if I have chores that depend on one another, finishing the first doesn't mean I have started the second, but starting on the second requires that I'm done with the first.